projects/knora-ui/src/lib/viewer/values/custom-regex.ts

```diff
@@ -0,0 +1,8 @@
+// @dynamic
+// https://github.com/ng-packagr/ng-packagr/issues/641
+
+export class CustomRegex{
+
+    public static readonly INT_REGEX = RegExp(/^-?\d+$/);
```

Comment:
I think it's fine like this `/^-?\d+$/`, see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp